app/assets/javascripts/views/history_view.js

```diff
@@ -5,41 +5,75 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addActivities', 'connectionError');
 
-    this.currentStory = {};
-
-    this.$el.html(this.template({id: 'history', name: I18n.t('projects.show.history')}));
+    this.$el.html(this.template({
+      id: 'history',
+      className:'historycolumn',
```

Comment:
So,  `.storycolumn` and `.historycolumn` have both the same style. However, `.storycolumn` is being used by the ProjectView, to identify columns the columns and instantiate all of them as `ColumnView`s. 

After the changes of this PR, `history_column` is there mainly to match the other columns classes pattern. Every column has a `${name}_column` class, and it is being used on feature tests to assert that the subject being tested is on the correct column. Do you think it would be okay to use `historycolumn`, on those tests related to the History column?